conda_lockfiles/create.py

```diff
@@ -40,7 +40,7 @@ def create_environment_from_lockfile(
     loader = Loader(lockfile)
     conda, pypi = loader.to_conda_and_pypi(environment=environment, platform=platform)
 
-    install_conda_records(conda, prefix)
+    install_conda_records(conda, str(prefix))
```

Comment:
`prefix` is passed directly to `PrefixData` which accepts `str | os.PathLike[str] | pathlib.Path`

Reply:
In `install_conda_records`, `prefix` is passed to `PrefixSetup` which only accepts a string. The type conversion could be deferred to that location.

Reply:
Ah I see, yeah thats out of scope